src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -70,75 +70,35 @@
  * Example:
  * </p>
  * <pre>
- * public void foo() throws Exception {
- *   int i = 0;
- *   while (i &gt;= 0) {
- *     switch (i) {
- *       case 1:
- *         i++;
- *       case 2: // violation, previous case contains code but lacks
- *               // break, return, yield, throw or continue statement
- *         i++;
- *         break;
- *       case 3: // OK
- *         i++;
- *         return;
- *       case 4: // OK
- *         i++;
- *         throw new Exception();
- *       case 5: // OK
- *         i++;
- *         continue;
- *       case 6: // OK
- *       case 7: // Previous case: OK, case does not contain code
- *               // This case: OK, by default the last case might not have statement
- *               // that transfer control
- *         i++;
+ * class Example1 {
+ *   public int foo() {
+ *     int i = 0;
+ *     while(i &gt;= 0) {
+ *       switch(i) {
+ *         case 1:
+ *           i++; // violation
+ *         case 2:
+ *           i++;
+ *           break;
+ *         case 3:
+ *           i++;
+ *           return;
+ *         case 4:
+ *           i++;
+ *           throw new Exception();
+ *         case 5:
+ *           i++;
+ *           continue;
+ *         case 6:
+ *         case 7:
+ *         case 8:
+ *           i++; // no break by design
```

Comment:
@romani. Do you want this comment to be changed to `// violation` because in Example1 we are considering config file with only `<module name="FallThrough"/>` ?

Reply:
Comment should stay. Because violation supposed to be on next line.